staff/daniel-toledo/cv/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel Toledo</title>
+</head>
+<body>
+    <header>
+        <h1>Curriculum Vitae Daniel Toledo</h1>
+        <h3>Renewable Energy Engineer and Future Javascript Programmer</h3>
+        <p>30th April 1992</p>
+        <p>Santa Catalina 18 1r, 08014 Barcelona, Spain</p>
+        <p><b>Tel :</b> +34 692461556</p>
+        <a href="mailto:daniel.toledomonfort@gmail.com">daniel.toledomonfort@gmail.com </a>
+        <dt><a href="https://www.linkedin.com/in/danieltoledomonfort/">Linkedin</a></dt>
+    </header>
+    <br>
+    <section>
+        <hr>
+        <h2><u>Experience</u></h2>
+        <br>
+        <ul>
+            <li>
+                <p><em>2017 Lisbon-Portugal</em></p>
+                <h3><b>Design Optimization of Mooring System for the WindFloat Technology</b></h3>
+                <h4>Principle Power Inc.</h4>
+
+                <p>
+                Intern  focused on the optimization of WindFloat’s mooring system, complying with certification 
+                entities requirements. Involved in FEED mooring system design for WFM project, consisting in 
+                a  24MW  (4  x  6MW)  offshore  wind  farm  in  Leucate  (France)  while  developing  MSc  Thesis:
+                ”Mooring  system  design  and  optimization  for  WindFloat  foundation  depending  on  water 
+                depths”
+                </p>
+            </li>
+        </ul>
+
+    </section>
+    <br>
+    <section>
+            <hr>
+            <h2><u>Education</u></h2>
```

Comment:
This is to style. The element was deprecated in HTML 4.01. (the element was used to define underlined text).
https://www.w3schools.com/tags/tag_u.asp